core/src/main/java/software/amazon/awssdk/retry/SdkDefaultRetryPolicies.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.retry;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import software.amazon.awssdk.RetryableException;
+import software.amazon.awssdk.http.HttpStatusCodes;
+
+public class SdkDefaultRetryPolicies {
```

Comment:
Is this a public or private class? From the name, I would have expected it to have retry policies in it.

Reply:
I'll rename it to something. Couldn't really figure out a good name.

This will be public as I don't see any reason not to expose these values to customers.